sdk/core/azure-core/azure/core/polling/_poller.py

```diff
@@ -102,7 +102,7 @@ def resource(self):
         return self._deserialization_callback(self._initial_response)
 
 
-class LROPoller(object):
+class LROPoller(Generic[PollingMethodType]):
```

Comment:
It means return type of `result` is `PollingMethodType`

Reply:
return type can be anything - typically a model. - can change it to PollingReturnType